src/kubernetes_cluster/spec/kubernetes_api/state_machine.rs

```diff
@@ -34,6 +34,8 @@ verus! {
 //
 // + Support more operations like List
 
+impl <K: CustomResourceView, E: ExternalAPI, R: Reconciler<K, E>> Cluster<K, E, R> {
```

Comment:
I am not against it, but what is the benefit of including all these spec methods into Cluster?

Reply:
Otherwise, these spec methods will have to carry generics.